js/wp-seo.js

```diff
@@ -1,8 +1,9 @@
 ;jQuery( function( $ ) {
-	var $document, addtag;
+	var $document, addtag, frame, metabox;
 
 	$document = $( document );
 	addtag    = document.getElementById( 'addtag' );
+	images    = jQuery( '.wp-seo-image-container' );
```

Comment:
because you're doing `jQuery( function( $ ) {` above, you can use `$` instead of `jQuery` here